### DISCUSSION TITLE
ISSUE: there are clashing file names in the book/img directory

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#21] [General]
For example 18-A.png and 18-a.png.  On MacOS file names are not case sensitive, so it would be best that either files with one case variant renamed to something distinct, or removed.

Here is what I get after git clone on a mac:

	modified:   book/img/18-A.png
	modified:   book/img/18-B.png
	modified:   book/img/18-C.png
	modified:   book/img/18-D.png
	modified:   book/img/18-E.png
	modified:   book/img/18-F.png
	modified:   book/img/18-G.png
	modified:   book/img/18-H.png
	modified:   book/img/18-I.png
	modified:   book/img/18-J.png
	modified:   book/img/18-K.png
	modified:   book/img/18-L.png
	modified:   book/img/18-M.png
	modified:   book/img/18-N.png
	modified:   book/img/18-O.png
	modified:   book/img/18-P.png
	modified:   book/img/18-Q.png
	modified:   book/img/18-R.png
	modified:   book/img/18-S.png
	modified:   book/img/18-T.png
	modified:   book/img/18-U.png
	modified:   book/img/18-V.png
	modified:   book/img/18-W.png
	modified:   book/img/18-X.png
	modified:   book/img/18-Y.png
	modified:   book/img/18-Z.png
	modified:   src/img/18-A.png
	modified:   src/img/18-B.png
	modified:   src/img/18-C.png
	modified:   src/img/18-D.png
	modified:   src/img/18-E.png
	modified:   src/img/18-F.png
	modified:   src/img/18-G.png
	modified:   src/img/18-H.png
	modified:   src/img/18-I.png
	modified:   src/img/18-J.png
	modified:   src/img/18-K.png
	modified:   src/img/18-L.png
	modified:   src/img/18-M.png
	modified:   src/img/18-N.png
	modified:   src/img/18-O.png
	modified:   src/img/18-P.png
	modified:   src/img/18-Q.png
	modified:   src/img/18-R.png
	modified:   src/img/18-S.png
	modified:   src/img/18-T.png
	modified:   src/img/18-U.png
	modified:   src/img/18-V.png
	modified:   src/img/18-W.png
	modified:   src/img/18-X.png
	modified:   src/img/18-Y.png
	modified:   src/img/18-Z.png

**Comment by aerickt** [comment]
lol that's crazy.
fixed: https://github.com/aerickt/lapwing-for-beginners/commit/9237ad4a45d06879c6e932763684c915aa70c172

Thank you.
